src/fs/inode_info.hh

```diff
@@ -85,11 +85,11 @@ struct inode_info {
         }
 
         // Deletes data vectors that are subset of @p data_range and cuts overlapping data vectors to make them
-        // not overlap. @p cut_data_vec_processor is called on each inode_data_vec (including parts of overlapped
+        // not overlap. @p data_vec_cutting_callback is called on each inode_data_vec (including parts of overlapped
         // data vectors) that will be deleted
         template<class Func>
-        void cut_out_data_range(file_range range, Func&& cut_data_vec_processor) {
-            static_assert(std::is_invocable_v<Func, inode_data_vec>);
+        void cut_out_data_range(file_range range, Func&& data_vec_cutting_callback) {
```

Comment:
A description of what data_vec_cutting_callback parameters are should be provided in the comment above.